### DISCUSSION TITLE
v1.1.2-beta.2

### DISCUSSION THREAD

**Discussion by thanksmister** [thanksmister/android-mqtt-alarm-panel#184] [Announcements]
This is a pre-release version of the application with an updated MQTT library. This version offers support for the Android 9 and above to use MQTT and resolve permissions.   This version supports a minimum SDK version of 21, it will not work on earlier versions of Android.

** Install the APK by downloading and clicking apk file. 

<hr /><em>This discussion was created from the release <a href='https://github.com/thanksmister/android-mqtt-alarm-panel/releases/tag/v1.1.2.2'>v1.1.2-beta.2</a>.</em>

**Comment by mreiling** [comment]
I am seeing problems with the alarm state and it being retained.  The logs on the alarm-panel keep showing state changes, to the same state, over and over and over.  Which in turn, keeps changing me from my dashboard screen back to the alarm screen.  If I delete the retained message on the MQTT Broker, the problem goes away.  Seems like it sees the retained message a new repeatedly.  Anyone else seeing this?
